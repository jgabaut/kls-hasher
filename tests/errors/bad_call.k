#!/bin/sh

./kls_hasher -bad
