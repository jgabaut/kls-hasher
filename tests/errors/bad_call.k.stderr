[ERROR]    kls_gulp_file_sized_to_kstr():  {File does not exist}.
try_kls_gulp_file_to_kstr():  kls_gulp_file_sized_to_kstr() failed with err {File does not exist}.
Failed KLS_GULP_FILE_KSTR()
